[WIKI] macOS installation

maltheism opened the pull request:
## Brief summary of changes

How to install Loris on macOS.
___

*Copied README.Mac.md content to new docs location for Loris installation.*

*Deleted README.Mac.md in root Loris directory.*

*Update url-links in two README.md associated with macOS install.*

ridz1208 commented on docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md:
```diff
@@ -1 +1,290 @@
+
+## Important Notes: 
+- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported for "production" as of v15.10
+- These instructions were last tested on LORIS version v20.*
```
redundant of the above lines

ridz1208 commented on docs/wiki/_DELETED/Install-Script-for-16.X.md:
```diff
@@ -27,7 +27,7 @@ For CentOS 6 systems a separate [[CentOS Readme|https://github.com/aces/Loris/bl
 ### Mac OSX
```
modifying files from `_DELETED` is not very useful, these files will be compt=letely deleted once the full wiki is released

johnsaigle commented on docs/deprecated_wiki/Installing-Loris.md:
```diff
@@ -24,12 +24,12 @@ For older versions of Ubuntu, we recommend adapting the 16.04 installation instr
 
 + View Ubuntu 16.04 instructions and adapt for Max OS X (recommended)
-+ View the [older instructions](https://github.com/aces/Loris/blob/master/README.Mac.md) for Mac OS X and adapt it for newer releases
++ View the [older instructions](https://github.com/aces/Loris/blob/master/docs/wiki/00%20-%20SERVER%20INSTALL%20AND%20CONFIGURATION/01%20-%20LORIS%20Install/Macintosh/README.md) for Mac OS X and adapt it for newer releases
```
You don't need to update deprecated files. They're just there to help with rewriting but shouldn't be modified.

johnsaigle commented on docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md:
```diff
@@ -1 +1,290 @@
+- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported for "production" as of v15.10
+- These instructions were last tested on LORIS version v20.*
+
+# How to install a developer instance of LORIS on macOS
```
Agreed with @ridz1208 that this should be moved to the `99` dev section so that it doesn't get mixed up with officially supported versions

johnsaigle commented on docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md:
```diff
@@ -1 +1,290 @@
+Nothing in this document is an official recommendation for LORIS but is only intended to be helpful!
+
+## Important Notes: 
+- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported for "production" as of v15.10
```
```suggestion
- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported as of v15.10
```

I don't think it was ever really supported for production but we used to support developer installs

johnsaigle commented on docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md:
````diff
@@ -1 +1,290 @@
+c. Type the following into the `ext-xdebug.ini` file you just created (Note: replace the `zend_extension` path with the correct one on your machine):
+```
+[xdebug]
+zend_extension="/usr/local/Cellar/php@7.2/7.2.14/pecl/20170718/xdebug.so"
````
```suggestion
zend_extension="/usr/local/Cellar/php@7.2/7.2.14/pecl/20170718/xdebug.so"
```

It might be good to replace this version info with `<your_php_version>` or something similar.

maltheism commented:
@ridz1208 @johnsaigle I removed redundancy, added version tag for php, and moved the README to a new location in Alternative Install. @ridz1208 these instructions are how I setup my macOS with LORIS for development. I believe @cmadjar used the instructions as well for setup on her macbook air.

ridz1208 commented:
@cmadjar if you used these could you give it a quick look and pass manual tests ?

ridz1208 commented:
@cmadjar @maltheism thank you,

I have no oppositions to this being merged.

johnsaigle commented on docs/wiki/99 - Developers/Alternative Install/LORIS on Macintosh/README.md:
```diff
@@ -172,9 +172,27 @@ sudo apachectl -k restart
+
+3) `make`
+
+4) `sudo cp modules/xdebug.so /usr/local/php/extensions` (create the the directory if it doesn't exit).
```
```suggestion
4) `sudo cp modules/xdebug.so /usr/local/php/extensions` (create the directory if it doesn't exit).
```

johnsaigle commented on docs/wiki/99 - Developers/Alternative Install/LORIS on Macintosh/README.md:
```diff
@@ -172,9 +172,27 @@ sudo apachectl -k restart
+
+Download the latest Xdebug source from https://xdebug.org
+
+After run the commands as follow in the folder of the source code:
```
What source code?

maltheism replied:
Thanks @johnsaigle, latest commit is more descriptive.

driusan commented on docs/wiki/99 - Developers/Alternative Install/README.md:
```diff
@@ -0,0 +1 @@
+Instructions for alternative installations of our software
```
I don't understand this file or why it's here.

johnsaigle replied:
Rida added a placeholder README in each of the new wiki directories to describe what they're for.

It can be deleted in this PR

maltheism replied:
@driusan @johnsaigle I removed it.

johnsaigle commented on docs/deprecated_wiki/Installing-Loris.md:
```diff
@@ -32,4 +32,4 @@ Mac OS X is no longer supported as of LORIS 15.10. You may do the following for
 
-**[[NEXT: (4) Imaging Setup|Imaging-Database]]**
\ No newline at end of file
+**[[NEXT: (4) Imaging Setup|Imaging-Database]]**
```
This is causing a rebase issue - since this is a deprecated file, I would suggest undoing your change to this file instead of rebasing.

driusan commented on docs/wiki/99 - Developers/Alternative Install/LORIS on Macintosh/README.md:
```diff
@@ -1,10 +1,6 @@
 
-## Important Notes: 
-- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported for "production" as of v15.10
-- These instructions were last tested on LORIS version v20.*
```
This remains an important note

maltheism replied:
Hi @driusan, I added it back with latest commit.

driusan replied:
It's still being removed in the diff

ridz1208 commented:
@maltheism can you move this to the DEVELOPER directory of the new wiki. otherwise its confusing when we say we dont support mac installs

maltheism commented:
@ridz1208 fixed.

maltheism commented:
@driusan first line states Mac isn't supported anymore.

driusan commented:
<img width="989" alt="Screen Shot 2563-07-15 at 11 06 46" src="https://user-images.githubusercontent.com/498329/87561914-59594d80-c68b-11ea-8dd4-d81514a461eb.png">

The important notes is still deleted

maltheism commented:
@driusan I added it back. Sorry I was confused.
